tests/unit/utils/sparqlhandler/test_csvw_inspector.py

```diff
@@ -256,3 +256,34 @@ def test_get_column_definitions_for_csv():
         value_url="http://example.com/value",
         virtual=True,
     )
+
+
+def test_multi_theme_and_keyword():
+    """
+    This test ensures that the 'select_catalog_metadata.sparql' will return the correct amount of
+    themes, keywords and landing pages(unique values and no duplications)
```

Comment:
It'd be helpful if you could stick the URL to the bug in here.

Reply:
yeah sure